schemas/data/record.schema.json

```diff
@@ -21,6 +21,14 @@
           "type": "string",
           "format": "uri-reference",
           "description": "A unique identifier for the record."
+        },
+        "xdm:eventList": {
```

Comment:
can we be explicit and name this "timeSeriesEvents" rather than eventList?

Reply:
@tcuevas-adobe I agree that the name can be improved. In practice the implementation will never be an actual `data/time-series` schema, but a union of a class (likely context/experienceevent(~99% of the time)) that has the behaviour `of data/time-series`

I wonder is there a better name, if not I will agree with `timeSeriesEvents`

Reply:
I'm also good with timeSeriesEvents